plugins/node/instrumentation-tedious/README.md

```diff
@@ -0,0 +1,58 @@
+# OpenTelemetry Tedious Instrumentation for Node.js
```

Comment:
Can you please also add the instrumentation to the list in the main repo's README?

Reply:
Done!